tools/pub_get_offline.py

```diff
@@ -127,6 +127,16 @@ def find_unlisted_packages():
   return unlisted
 
 
+def delete_config_files():
+  # Find all package_config.json that are not under version control.
+  gitcmd = ['git', 'ls-files', '-o', '**/.dart_tool/package_config.json']
+  files_to_delete = subprocess.check_output(
+      gitcmd, cwd=ENGINE_DIR, stderr=subprocess.STDOUT, text=True
+  ).splitlines()
+  for file in files_to_delete:
+    os.remove(os.path.join(ENGINE_DIR, file))
```

Comment:
should we log what we're deleting? or am I being paranoid?

Reply:
I'll add a logging statement.

Reply:
Done.